src/worker.js

```diff
@@ -35,6 +34,10 @@ var __performance_now_clock_drift = 0;
 // Cannot use console.log or console.error in a web worker, since that would risk a browser deadlock! https://bugzilla.mozilla.org/show_bug.cgi?id=1049091
 // Therefore implement custom logging facility for threads running in a worker, which queue the messages to main thread to print.
 var Module = {};
+#if EXPORT_ES6
```

Comment:
i think we should also emit these lines if MODULARIZE as we do assign to them there as well (line 144/158), and for the same reason?

Reply:
The difference is that in normal (aka sloppy) JS you can define things without var/const/let. So defining `PThread` like `PThread = Module['PThread'];` is perfectly valid. However this is not valid inside an ES6 module as that enforces strict mode. That's why I had to declare them here.
I can change `#if EXPORT_ES6` to `#if MODULARIZE` if you want, but there may be a reason that it was initially done like that and I didn't want to break existing modularize code.